modules/authorization-manager/src/main/java/org/opencastproject/authorization/xacml/manager/endpoint/AbstractAclServiceRestEndpoint.java

```diff
@@ -223,21 +306,40 @@ public Response deleteAcl(@PathParam("aclId") long aclId) throws NotFoundExcepti
 
   @POST
   @Path("/apply/episode/{episodeId}")
-  @RestQuery(name = "applyAclToEpisode", description = "Immediate application of an ACL to an episode", returnDescription = "Status code", pathParameters = { @RestParameter(name = "episodeId", isRequired = true, description = "The episode ID", type = STRING) }, restParameters = {
-          @RestParameter(name = "aclId", isRequired = false, description = "The ID of the ACL to apply. If missing the episode ACL will be deleted to fall back to the series ACL", type = INTEGER)}, responses = {
+  @RestQuery(
+      name = "applyAclToEpisode",
+      description = "Immediate application of an ACL to an episode",
+      returnDescription = "Status code",
+      pathParameters = {
+          @RestParameter(name = "episodeId", isRequired = true, description = "The episode ID", type = STRING)
+      },
+      restParameters = {
+          @RestParameter(
```

Comment:
This formatting doesn't match the changes above :)

Reply:
Again, here, unlike above, the `@RestParameter` doesn't fit into one line, therefore it's broken into multiple ones.